peribolos/bots.md

```diff
@@ -0,0 +1,40 @@
+# Knative Automation Bots
+
+The peribolos configuration in this directory is used to configure users and groups in GitHub.
+Among those accounts configured are several bot accounts that are used for automated tasks
+in the Knative testing and release infrastructure. Those user accounts are listed here:
+
+- knative-automation
+- knative-metrics-robot
+- knative-prow-robot
+- knative-prow-releaser-robot
+- knative-prow-updater-robot
+- knative-test-reporter-robot
+
+## Knative Automation [@knative-automation](https://github.com/knative-automation)
+
+- Synchronizes GitHub actions and automated workflows.
+- Updates snapshot dependency versions between projects.
+- Updates common dependencies across all projects.
+- Synchronizes GitHub configuration across all projects.
+
+## Knative Metrics Robot [@knative-metrics-robot](https://github.com/knative-metrics-robot)
+
+- No recent activity
+
+## Knative Prow Robot [@knative-prow-robot](https://github.com/knative-prow-robot)
+
+- Used to synchronize cherry picked commits across release branches
```

Comment:
Probably easiest to have this bot have CLA signoff, too, though the `/cherrypick` command should also be bringing along the original author info which could be used to verify CLA status.

Reply:
The knative-prow-robot bot account is also the owner of the Knative Prow GitHub app - https://github.com/apps/knative-prow, not sure if it's worth mentioning here though.